static/js/containers/ProfileFormContainer.js

```diff
@@ -98,6 +98,12 @@ class ProfileFormContainer extends React.Component {
         dispatch(clearProfileEdit(username));
       });
     } else {
+      this.setState({}, () => {
```

Comment:
Good idea, we are, after all, using a side-effect of `setState` and not actually, you know, setting any state, haha.